File: AndroidStudio/AlertDialogSample/app/src/main/java/to/msn/wings/alertdialogsample/MainActivity.java
package to.msn.wings.alertdialogsample;

import androidx.appcompat.app.AppCompatActivity;
import androidx.fragment.app.DialogFragment;

import android.app.Dialog;
import android.os.Bundle;
import android.view.View;

public class MainActivity extends AppCompatActivity
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
    }{


        public void btn_onClick(View view){
        DialogFragment dialog = new MyDialogFlagment();
        dialog.show(getSupportFragmentManager(),"dialog_");
    }
}

File: Android_MyApp/ImpartialReferee/app/src/main/java/to/msn/wings/impartialreferee/MainActivity.kt
package to.msn.wings.impartialreferee

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.view.View
import androidx.fragment.app.DialogFragment
import androidx.fragment.app.FragmentManager
import androidx.viewpager.widget.ViewPager

class MainActivity : AppCompatActivity() {

    /**
     * Activityが立ち上がった時
     */
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        // -----------------------------------------------------------------------//

        //ViewPagerにアダプタを設定
        val pager : ViewPager = findViewById(R.id.pager)
        pager.adapter = PageAdapter(supportFragmentManager)
    }

    fun onClickSelectButton(view: View){
        val dialogFragment = SelectDialogFragment()
        dialogFragment.show(supportFragmentManager,"aa")
    }
}
